http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java

```diff
@@ -430,6 +430,7 @@ public void constraintsExist() throws JsonProcessingException {
     }
 
     @Test
+    @Tag("QUARKUS-3672")
     @Tag("https://github.com/quarkusio/quarkus/pull/36664")
```

Comment:
Maybe we can remove the tag with upstream issue as now Quarkus link it together.

Reply:
oki